common/djangoapps/entitlements/api/v1/views.py

```diff
@@ -257,7 +258,13 @@ def create(self, request, uuid):
     def destroy(self, request, uuid):
         """
         On DELETE call to this API we will unenroll the course enrollment for the provided uuid
+
+        If is_refund parameter is provided then unenroll the user, set Entitlement expiration, and issue
+        a refund
         """
+        is_refund = True if request.query_params.get('is_refund', 'false') == 'true' else False
```

Comment:
You can just do:
```
is_refund = request.query_params.get('is_refund', 'false') == 'true'
```

Reply:
I will make this change in a follow on PR.